src/main/java/com/p6spy/engine/event/JdbcEventListener.java

```diff
@@ -435,4 +435,27 @@ public void onAfterRollback(ConnectionInformation connectionInformation, long ti
   public void onAfterStatementClose(StatementInformation statementInformation, SQLException e) {
   }
 
+
+  /**
+   * this callback method is executed before calling {@link Connection#setAutoCommit(boolean)}
```

Comment:
Please make it consistent with other javadoc - capitalize first word and add line break before parameters.

Reply:
done